homeassistant/components/overkiz/climate_entities/atlantic_pass_apc_heating_zone.py

```diff
@@ -49,7 +49,15 @@
     OverkizCommandParam.INTERNAL_SCHEDULING: PRESET_HOME,
 }
 
-PRESET_MODES_TO_OVERKIZ = {v: k for k, v in OVERKIZ_TO_PRESET_MODES.items()}
```

Comment:
I changed this, because multiple there was multiple identical values, it was not kiss enough